updater/scripts/git-download.sh

```diff
@@ -15,13 +15,20 @@ function ghe_greater_equal () {
 
 if ghe_greater_equal "2.11.0" ; then
     LOG_FILE="/var/log/github-audit.log"
+    # The "github-audit.log" log file introduced in GHE 2.11.0 is only rolled
+    # once a week. This was reported as bug and is likely fixed in an upcoming
+    # version. In the meantime we grep for all log entries that have been
+    # written yesterday.
+    GREP_YESTERDAY="grep -F '$(date --date='yesterday' +'%b %d')'"
```

Comment:
Maybe, we could add something like `# TODO: remove workaround once GitHub Enterprise is patched` so that we clean this up as soo as this workaround isn’t necessary anymore?

Reply:
Well, I don't think we can remove it because GHE 2.11.0 - 2.11.x will always have this bug.